core/arch/arm/plat-ls/main.c

```diff
@@ -61,13 +72,60 @@ static void main_fiq(void)
 	panic();
 }
 
+void plat_cpu_reset_late(void)
+{
+	static uint32_t cntfrq;
```

Comment:
Thanks for the careful review.
correct, it should be in __early_bss;
however, the secondary cores use it before core0 enters main(), so it actually works, although it shall be placed in __early_bss.